templates/layout.html

```diff
@@ -22,6 +22,21 @@
 	<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
 	<meta name="theme-color" content="#2b2f33" /> 
 	<meta name="format-detection" content="telephone=no">
+  {% block meta %}
+  {% if page_title %}
+  <title>{{ brand.name }} :: {{ page_title }}</title>
+  {% else %}
+  <title>{{ brand.name }}</title>
+  {% endif %}
+  <meta name="twitter:description" content="{% firstof page_description brand.description %}" />
+  <meta property="og:description" content="{% firstof page_description brand.description %}" />
+  <meta name="description" content="{% firstof page_description brand.description %}">
+
```

Comment:
The canonical on [line 13](https://github.com/appu1232/guyamoe/blob/master/templates/layout.html#L13) can probably be changed here as well. 

This does bring up the issue that appu mentioned in his other comment regarding `root_domain` being globally available; it doesn't make sense to manually pass it around, but it's probably useful to have available in all our templates. 

--

One thing that I noticed is that we add the trailing slash in the context as well as the template, so it doubles up for some pages like `/about` and `/pages`. We should probably change that, but it's not a huge deal and also probably out of scope here.